CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## v1.7.0 (06/15/2020)
+
+**Common**
+
+* Corrected various formatting to be compliant with the coding style.
+* Fixed the issue with `olp::client::TaskContext`. Now, it does not trigger cancellation after a successful operation.
+* Added the `default_cache_expiration` added to `olp::client::OlpClientSettings`. Now, you can can control the time how long the downloaded data is considered valid. Once the data has expired it will not be returned to the user upon requests and gradually removed from the cache when new data is added.
+* Added the `OLP_SDK_DISABLE_DEBUG_LOGGING` CMake option. When enabled, the SDK does not print any `TRACE` or `DEBUG` level log messages.
+* Added the `compression` setting to `olp::cache::CacheSettings`. You can now pass the flag to the underlying database engine to enable or disable the data compression.
+* Added the `Compact` API to `olp::cache::DefaultCache`. It is used to optimize the underlying mutable cache storage.
+* Changed the `olp::cache::DefaultCache` read operation. As a result, the number of memory allocation decreased, and performance increased.
+* Added support for the HTTP `OPTIONS` request in `olp::http::Network`.
+* The LRU eviction now evicts expired elements first. The eviction time decreased.
+* Removed the retry code from `olp::http::NetworkCurl`. Now, retries are performed by `olp::client::OlpClient`.
+* Added a prewarm helper method to `olp::client::OlpClientSettingsFactory`. This can be used to perform DNS prefetch and TLS preconnect for certain URLs that are widely used by the clients, e.g. account and lookup.
+* Enabled certificate validation in `olp::http::NetworkCurl`.
+* Possible crash fixed in `olp::cache::DefaultCache`, which could occur during initialization in the spurious case when the expiration value was not null terminated.
+
+**olp-cpp-sdk-authentication**
+
+* The `olp::authentication::TokenProvider` class now handles the concurrent  by blocking all requesters until the already triggered request completes. This will make sure that only one token is requested at the same time.
```

Comment:
```suggestion
* The `olp::authentication::TokenProvider` class now handles the concurrent by blocking all requesters until the already triggered request completes. This will make sure that only one token is requested at the same time.
```